docs/_documentations/metrics-dashboard.md

```diff
@@ -103,9 +103,14 @@ If profiling markers do not appear, check to see if your project and load run co
 
 ## Interpreting the annotation tooltip
 
-After you start the load test and the test completes, view the source code. Theia displays an annotation tooltip with a specified message in a similar format to the following example:
+After you start the load test and the test completes, view the source code. Theia and VS Code display an annotation tooltip with a specified message in a similar format to the following example:
 
-![image of Theia annotation tooltip](images/theia-annotation-tooltip.png)
+`Function app.get() was the running function in 0.58% of samples.`
```

Comment:
i think this whole block should be in a fenced code block (`````) so appears as one piece

Reply:
OK, I've changed it to a code block. 👍